webapp/tests/Unit/Controller/Jury/ContestControllerTest.php

```diff
@@ -191,7 +191,7 @@ public function testCheckAddEntityAdmin(): void
 
     public function testUnlockJudgeTasks(): void
     {
-        // First, check that adding a submission creates a queue task and 3 judge tasks
+        // First, check that adding a submission creates a queue task and 4 judge tasks.
```

Comment:
Why do we suddenly start with 4 judge tasks? I assume you're correct but I don't see how moving away the fixture also changes this test.

Reply:
Because the problem has now one more testcase (1 sample, 3 secret). Another sign that we should make the unit tests actual unit tests.